### PR TITLE
chore(card): update stories for card

### PR DESCRIPTION
**Pull request by spadidapelli**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)
- [ ] If applicable, have a visual design approval

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number:CDE-1522

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: .storybook/stories/card/card.stories.ts [hunk 1]
```diff
@@ -65,8 +66,6 @@ const defaultParameters: Parameters = {
...
 };
 
-const variants: Parameters[] = buttonTypes.map(buttonType => ({
-  buttonType,
-}));
-
-setupStorybook([], defaultStory, defaultParameters, variants);
+export const Default: StoryObj = {
+  render: cardTemplate,
+};
```
Comment (dtsanevmw) [inline .storybook/stories/card/card.stories.ts:71]: Should be named different from 'Default' before we finish all the refactorings during current condition that ignores `--default` stories for visual testing.
```suggestion
export const Card: StoryObj = {
  render: cardTemplate,
};
```

File: .storybook/stories/card/card.stories.ts [hunk 2]
```diff
@@ -4,12 +4,11 @@
...
 
-import { Parameters } from '@storybook/addons';
-import { Story } from '@storybook/angular';
+import { Story, StoryObj } from '@storybook/angular';
 
-import { setupStorybook } from '../../helpers/setup-storybook.helpers';
+const buttonTypes = ['btn-primary', 'btn-outline', 'btn-link'];
 
-const defaultStory: Story = args => ({
+const cardTemplate: Story = args => ({
```
Comment (dtsanevmw) [inline .storybook/stories/card/card.stories.ts:11]:
```suggestion
const CardTemplate: Story = args => ({
```

**Review by dtsanevmw — CHANGES_REQUESTED** [review]
One small change to be made on the name and if you can put the template after default export.

File: .storybook/stories/card/card.stories.ts (hunk 1 above)
Comment (spadidapelli) [reply .storybook/stories/card/card.stories.ts]: updated the story as per above name

**Pull request merged**